deploy/medulla/values.yaml

```diff
@@ -0,0 +1,21 @@
+---
+shell:
+  replicaCount: 1
+  image:
+    repository: shell
+    pullPolicy: Always
+    tag: "1.0.0"
+
+identity:
+  replicaCount: 1
+  image:
+    repository: identity
+    pullPolicy: Always
+    tag: "1.0.0"
+
+documentation:
+  replicaCount: 1
+  image:
+    repository: documentation
```

Comment:
Looks like were missing the registry values here, meaning this is pulling from docker hub correct? This wont work if we plan on releasing Iron Bank images / GHCR.

Containers are broken down into a few sections:

Registry: ghcr.io
Repository: recro/medulla
Image: documentation
Tag: 0.1.0

Full URL: ghcr.io/recro/medulla/documentation:0.1.0

Reply:
Thank you for the input. I will work on adding this.